sdk/src/client/secret/mod.rs

```diff
@@ -235,17 +240,13 @@ impl SecretManage for SecretManager {
     ) -> crate::client::Result<Vec<Address>> {
         match self {
             #[cfg(feature = "stronghold")]
-            Self::Stronghold(secret_manager) => {
-                secret_manager
-                    .generate_addresses(coin_type, account_index, address_indexes, options)
```

Comment:
Why are we wrapping in Ok here?

Reply:
because I'm using `?` to convert the error (which is not a ClientError anymore) so we need to rewrap it